src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -148,4 +158,16 @@ private boolean shouldFormaliseResponseAcceptance(Response response, ClaimantRes
             && !ResponseUtils.isResponseStatesPaid(response)
             && !ResponseUtils.isResponsePartAdmitPayImmediately(response);
     }
+
+    private boolean isRejectResponseWithMediationForMIReporting(Claim claim, ClaimantResponse claimantResponse) {
```

Comment:
Claim passed is used only to retrieve response. may be we can send the response to this method instead of whole claim

Reply:
It is being used similarly in other methods.